MaxText/configs/models/deepseek2-16b.yml

```diff
@@ -40,8 +40,6 @@ kv_lora_rank: 512
 qk_nope_head_dim: 128
 qk_rope_head_dim: 64
 v_head_dim: 128
-# RoPE
-rope_type: "yarn"
```

Comment:
I feel this remove the flexibility of rope in models. For instance, how we apply llama3.1 type to a new future deepseek model?

Reply:
I reverted this change as discussed offline. Thanks for your insights!